packages/react/src/Header/Header.tsx

```diff
@@ -84,7 +96,7 @@ const HeaderRoot = forwardRef(
             {/* The branding section is recreated here, to make sure the page menu breaks at the right spot */}
             <div aria-hidden className="ams-header__branding ams-header__branding--hidden">
               <div className="ams-header__logo-link">
-                <Logo brand={logoBrand} />
+                <LogoComponent logoBrand={logoBrand} responsiveLogo={responsiveLogo} />
```

Comment:
The nice thing about the logo svg was that you could've just hidden the text path.

Reply:
That doesn’t make the SVG any narrower. How would you approach that?

Reply:
I've tried to use CSS modules to do this:
```css
@container (max-width: 36rem) {
    .header div:first-child > a {
        max-inline-size: 12px;
    }
    .header div:first-child > a svg path:nth-child(1) {
        opacity: 0;
    }
}
```

Reply:
My problem with the breakpoint is that you do not anticipate a long or short brandname. So a tablet size with a short title will hide the logo text way too soon.

I would prefer a named container for the brandname but a grid stack has been used so it won't work.

Reply:
> The dimensions are pretty much fixed

They aren't though. The logo grows from 40 to 56 px in height. I'm not sure your approach works with a fluid logo.